svir/dialogs/load_losses_by_asset_as_layer_dialog.py

```diff
@@ -93,11 +97,13 @@ def on_rlz_or_stat_changed(self):
         self.dataset = self.npz_file[self.rlz_or_stat_cbx.currentText()]
         self.taxonomies = numpy.unique(self.dataset['taxonomy']).tolist()
         self.populate_taxonomy_cbx(self.taxonomies)
-        # discarding 'asset_ref', 'taxonomy', 'lon', 'lat'
-        self.loss_types = self.dataset.dtype.names[4:]
-        self.populate_loss_type_cbx(self.loss_types)
         self.set_ok_button()
 
+    def populate_out_dep_widgets(self):
+        self.populate_rlz_or_stat_cbx()
+        self.populate_loss_type_cbx(self.loss_types)
+        self.show_num_sites()
+
```

Comment:
This 4 lines are duplicated. You can just turn `populate_out_dep_widgets` into an utility function instead of a method.